src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -18,7 +18,9 @@ class HoldRequest extends React.Component {
     super(props);
 
     this.state = _extend({
-      delivery: false,
+      delivery: (this.props.deliveryLocations.length > 0) ?
+        this.props.deliveryLocations[0]['@id'].replace('loc:', '') : 'edd',
+      checkedLocNum: (this.props.deliveryLocations.length > 0) ? 0 : -1,
```

Comment:
I didn't understand at first why you did `0` or `-1` but then it made sense. Can you write some comments about it to make it easier to understand?